spanner/batch.go

```diff
@@ -212,13 +213,14 @@ func (t *BatchReadOnlyTransaction) partitionQuery(ctx context.Context, statement
 
 	// prepare ExecuteSqlRequest
 	r := &sppb.ExecuteSqlRequest{
-		Session:        sid,
-		Transaction:    ts,
-		Sql:            statement.SQL,
-		Params:         params,
-		ParamTypes:     paramTypes,
-		QueryOptions:   qOpts.Options,
-		RequestOptions: createRequestOptions(qOpts.Priority, qOpts.RequestTag, ""),
+		Session:          sid,
+		Transaction:      ts,
+		Sql:              statement.SQL,
+		Params:           params,
+		ParamTypes:       paramTypes,
+		QueryOptions:     qOpts.Options,
+		RequestOptions:   createRequestOptions(qOpts.Priority, qOpts.RequestTag, ""),
+		DataBoostEnabled: qOpts.DataBoostEnabled,
```

Comment:
Add a one line comment:
`If this is for a partitioned query and this field is set to true, the request will be executed via Spanner independent compute resources. `

Reply:
i have added it under QueryOptions
https://github.com/googleapis/google-cloud-go/pull/8152/files#diff-98c680ad0e9a6842d77932c6b37ee80a322d5b5e462a26a4f0154844ea4e8513R169